js/exporter/jspdf/export.js

```diff
@@ -68,11 +67,8 @@ export const Export = {
                 const styles = dataProvider.getStyles();
                 const dataRowsCount = dataProvider.getRowsCount();
 
-                if(tableWidth) {
-                    autoTableOptions.tableWidth = this.convertPixelsToPoints(tableWidth);
-                }
-
-                this.setColumnWidths(autoTableOptions, columns, keepColumnWidths);
+                const pdfColumnWidths = this.getDefaultPdfColumnWidths(autoTableOptions.tableWidth, dataProvider.getColumnsWidths());
```

Comment:
imho 'getDefaultPdfColumnWidths' should work without 'this'

Reply:
checked, didn't work without 'this'
in transpiled code:
```
_this = this
...
  return new Promise(function (resolve) {
    dataProvider.ready().done(function () {
...
_this.getDefaultPdfColumnWidths(...)
```